assets/css/style.css

```diff
@@ -321,3 +321,21 @@ html[dir=rtl] .pull-right:not(#rtlChecker) {
 html[dir=rtl] .pull-left:not(#rtlChecker) {
     float: right !important;
 }
+
+/* Styling for the back-to-top button */
+#backToTop {
+    display: none;
+    position: fixed;
+    z-index: 999;
+    right: 20px;
+    bottom: 20px;
+    opacity: 0.8;
+}
+
+#backToTop:hover {
+    opacity: 1;
+}
+
+.fa-arrow-up {
```

Comment:
Use something like `#backToTop .fa-arrow-up`. This needs to not affect all the arrows.